Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -591,14 +591,21 @@ private Task ForceCacheFlushAsync(OperationContext context)
                     try
                     {
                         await Task.Yield();
-                        await _inMemoryCache.FlushAsync(context);
-                        return BoolResult.Success;
+                        return Result.Success(await _inMemoryCache.FlushAsync(context));
                     }
                     finally
                     {
                         Interlocked.Exchange(ref _cacheUpdatesSinceLastFlush, 0);
                         ResetFlushTimer();
                     }
+                }, extraEndMessage: maybeStatistics => {
+                    if (!maybeStatistics.Succeeded)
+                    {
+                        return string.Empty;
+                    }
+
+                    var statistics = maybeStatistics.Value;
+                    return $"Persisted={statistics.Persisted} Leftover={statistics.Leftover} Growth={statistics.Growth} FlushingTimeMs={statistics.FlushingTime.TotalMilliseconds} CleanupTimeMs={statistics.CleanupTime.TotalMilliseconds}";

```

Comment:
Minor: I think I would prefer FlushingDuration=1234ms (the same for all the other durations.